### DISCUSSION TITLE
Edit TableViewTextColumn

### DISCUSSION THREAD

**Discussion by JLBLU** [w-ahmad/WinUI.TableView#73] [Feedback Wanted]
Hi Ahmad,
I have a tableview that can go from readonly state to input allowed state depending on a given action.
When the tableview becomes "input allowed", I have sometimes some difficulties to enter in the field : double click has no effect. It seems to work if I click in my cell immediately after having moved cursor from a row to another row. But, if I wait a few seconds, click has no more effect and my cell seems to be readonly. And it works only if I click in a corner of my cell !??Strange ... If you have any idea ...
best regards 

                    <controlsTV:TableViewTextColumn
                            Header="Nom Dossier" 
                            Binding="{Binding DocFolderName, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"/>

even if I add "IsReadonly=..."

                    <controlsTV:TableViewTextColumn
                            Header="Nom Dossier" 
                            Binding="{Binding DocFolderName, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
                            IsReadOnly="{Binding ElementName=tvDkSettingsDocFolders, Path=IsReadOnly}"/>

**Comment by w-ahmad** [comment]
Looks like this problem is just like the one in #97.

**Comment by w-ahmad** [comment]
Related issue (#97) is fixed.
